### ISSUE TITLE
Recording photo optional

### ISSUE THREAD

**Issue by stevenbird** [aikuma/aikuma#477]
The recording photo should be optional. A recording can be finalised without a photo.

**Comment by Lingomat** [comment]
This is fair enough, but it raises a critical UI issue. We are going down a road that allows almost non-existant metadata on primary items. We need to highly encourage users to enrich them by some other mechanism since the wizard-approach is obsolete.

I'm going to create a proposal on the Wiki. <groans all round>.

**Comment by stevenbird** [comment]
When forced to take a photo against their will, users photograph random things instead, like the floor.

**Comment by Lingomat** [comment]
Yes, I get that, it's still creating a big problem. I've knocked up the proposal on the Wiki, I'm just going to do a couple of mocks.

**Comment by LaurenGawne** [comment]
I know it's not the solution that you want, but could you use an
autogenerated image of some kind - a bit like the Twitter egg pics?

**Comment by Lingomat** [comment]
I don't mind if we don't have a photo for some reason, I would just like to have a mechanism to encourage users to take a photo (and perform other actions) for any given item. 

I'm totally on board with the purpose of cutting the wizard-based approach so we get people out of recording activities and back to the main screen quickly. So that they have the freedom to choose their next task without being locked into an inflexible set of actions.

However this has serious ramifications for unsupervised metadata collection, namely we'll just get less of it and I think we need more, not less. Much easier to what I show in the mockups here: https://github.com/aikuma/aikuma/wiki/UI-proposal

Be interested to hear your thoughts on that.

**Comment by floere** [comment]
Why is the wizard based approach obsolete? What was the reasoning behind it?

**Comment by floere** [comment]
And re: the optional photo taking. Another good reason is that it may simply be night at the time a recording is made. See Reitmaier et al. 2011, page 666, for an example.

**Comment by Lingomat** [comment]
Reasoning: It locks the user into an inflexible sequence of steps via many activities in a chain. You can't quickly begin another recording session, and in relation to this issue, it mandates several steps. The problem is compounded by Aikuma's lack of a breadcrumb trail so you have a long sequence of screens that look almost identical. Granted that could be addressed but I think wizards are best deployed for essential complex tasks where a wizard is explaining each step in detail.

Steven and I, at least, thought a better approach is to allow the user to perform the tasks they wish to. I'm still convinced this is a much better process but it needs the feedback process to encourage users to fill in essential metadata.

The photo issue isn't the best example since it really is optional. Consider this instead: At the moment after the user has made a recording, there are no users associated. The recording *cannot* be used for anything because we cannot demonstrate informed consent. I'm raising that general concern, not specifically with photos.

**Comment by stevenbird** [comment]
We need to delay this until we have a way to add a photo later.
